packages/build/webpack/constants.js

```diff
@@ -9,16 +9,26 @@ const path = require('path');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 
-// This path assumes the enlistment in labkey-ui-components is a sibling of the root of the LabKey enlistment.
-// TODO figure out how we can make this more flexible and get an absolute path from the dev based on their setup
-const labkeyUIComponentsRelPath = (lkModuleContainer ? "../../../../../" : "../../../../") + "labkey-ui-components/packages/components";
-const labkeyUIComponentsPath = process.env.LINK ? path.resolve(labkeyUIComponentsRelPath) : path.resolve("./node_modules/@labkey/components");
-console.log("Using @labkey/components path: " + labkeyUIComponentsPath);
+// Conditionalize the path to use for the @labkey packages based on if the user wants to LINK their labkey-ui-components repo.
+// NOTE: the LABKEY_UI_COMPONENTS_HOME environment variable must be set for this to work.
+let labkeyUIComponentsPath = path.resolve("./node_modules/@labkey/components");
+let freezerManagerPath = path.resolve("./node_modules/@labkey/freezermanager");
+if (process.env.LINK) {
+    if (process.env.LABKEY_UI_COMPONENTS_HOME === undefined) {
+        console.log("ERROR: You must set your LABKEY_UI_COMPONENTS_HOME environment variable in order to link your @labkey packages.")
```

Comment:
console.error?

Reply:
I had it as console.error, but it didn't display any differently in the webpack log output and it didn't stop execution of the command. If anyone knows of how to stop a webpack command execution, then we should add that here.

Reply:
Did you try throwing?

Reply:
Yup, throw does the trick. Thanks. I'll put that in the next @labkey/build version.